float32 lift_extension
---
bool success